ci-operator/config/openshift/release/openshift-release-master__nightly-4.19.yaml

```diff
@@ -75,6 +75,10 @@ base_images:
     name: rosa-aws-cli
     namespace: ci
     tag: latest
+  tests-private:
```

Comment:
What's the purpose for adding this "tests-private" stanza? I don't see it used anywhere?